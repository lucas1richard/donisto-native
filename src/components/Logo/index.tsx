import * as React from 'react';
import { View } from 'react-native';
import theme from 'theme/iftheme';

class Logo extends React.Component {
  render() {
    return (
      <View style={{ alignItems: 'center' }}>
        <View
          style={{
            height: 80,
            width: 80,
            borderRadius: 80,
            borderColor: theme.white,
            borderWidth: 4
          }}
        />
      </View>
    );
  }
}

export default Logo;
