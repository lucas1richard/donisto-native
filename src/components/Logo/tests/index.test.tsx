import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Logo from '..';

describe('<Logo />', () => {
  const renderComponent = () => toJson(shallow(<Logo />));
  it('renders', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});