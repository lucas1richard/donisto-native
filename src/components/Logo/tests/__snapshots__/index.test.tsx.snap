// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Component
    style={
      Object {
        "borderColor": "#FFFFFF",
        "borderRadius": 80,
        "borderWidth": 4,
        "height": 80,
        "width": 80,
      }
    }
  />
</Component>
`;
